src/frontend/screens/Settings/components/OtherSettings/GameMode.tsx

```diff
@@ -48,7 +53,7 @@ const GameMode = () => {
     <div className="toggleRow">
       <ToggleSwitch
         htmlId="gamemode"
-        value={useGameMode}
+        value={false}
```

Comment:
is this intentional?

Reply:
nope. dunno why that was changed. my mistake. fixed now. nice catch